woocommerce/woocommerce-ios
# [Store onboarding] Polish store setup card

Part of: #8907


## Description

Polishes the store setup card with the following changes

- Hide the divider for the last task
- Show a chevron icon (`>`) at the trailing edge for each task 
- Show a progress bar at the top with different alignments in the collapsed/expanded state
- Show a "View all (#)" at the bottom to see the expanded state
- New icon for launch store task

## Testing instructions
- Login if needed
- Launch the app and navigate to Dashboard screen 
- Check the "Store setup" task list in collapsed mode.
- Tap on any task to open the expanded task list.
- You can compare the following design with the screens

Design for,
- collapsed state - VyLr7LvKodHE4kINfBE7Lw-fi-1644%3A82231
- Expanded state - VyLr7LvKodHE4kINfBE7Lw-fi-1644%3A82105

**Pending UI work from #8907**
- [ ] Ensure every icon & checkmark icon is the same size
- [ ] Add padding with a different background color above & below the onboarding list

## Screenshots
| | Expanded | Collapsed |
| - | - | - |
| Light | ![Simulator Screen Shot - iPhone 14 - 2023-02-23 at 19 28 58](https://user-images.githubusercontent.com/524475/220928984-b238c127-9d20-4c2c-a0af-af4c160f6064.png) | ![Simulator Screen Shot - iPhone 14 - 2023-02-23 at 19 29 01](https://user-images.githubusercontent.com/524475/220929016-722b5252-01e6-4aa6-b03d-7a6404f09cda.png) |
| Dark | ![Simulator Screen Shot - iPhone 14 - 2023-02-23 at 19 33 04](https://user-images.githubusercontent.com/524475/220929990-48d0bbfb-42a0-43ae-885f-af2654a902a1.png) | ![Simulator Screen Shot - iPhone 14 - 2023-02-23 at 19 33 00](https://user-images.githubusercontent.com/524475/220929972-ae238b82-7cf1-4964-9a6a-3d9f2844b20d.png) |

---
